CONTRIBUTING.md

```diff
@@ -15,6 +15,19 @@ Before sending your pull requests, ensure that you follow this checklist:
 
 * Ensure that [unit tests](CONTRIBUTING.md#unit-tests) pass. Include logs from tests as attachments to the pull request.
 
+* Ensure that corresponding maintainer GitHub team is assigned to the PR review.
+
+  | GitHub team name | Description |
+  :-----------|:------------|
+  | @oneapi-src/onemkl-maintain  | All oneMKL maintainers |
+  | @oneapi-src/onemkl-arch-write | oneMKL Architecture maintainers |
+  | @oneapi-src/onemkl-blas-write | oneMKL BLAS maintainers |
+  | @oneapi-src/onemkl-dft-write | oneMKL DFT maintainers |
+  | @oneapi-src/onemkl-lapack-write | oneMKL LAPACK maintainers |
+  | @oneapi-src/onemkl-rng-write | oneMKL RNG maintainers |
+  | @oneapi-src/onemkl-sparse-write | oneMKL Sparse Algebra maintainers |
+  | @oneapi-src/onemkl-vm-write | oneMKL Vector Math maintainers |
```

Comment:
I think "oneMKL" term usage should be avoided or at least specify that it's about oneMKL **Interfaces** not a oneMKL product.

Reply:
@Rbiessy, it's a good question, I need to refer to the table in both places, but I don't really like this duplication, let me think how to keep only one table, but make it available in both places without duplicating the table itself.

@itopinsk, good point, while RFC about re-naming https://github.com/oneapi-src/oneMKL/pull/564 is in progress, it makes sense to use `oneMKL Interfaces` to avoid confusion with Intel oneMKL

Reply:
I've added missed `Interfaces` to the group description and removed table duplicate from README.md, instead we refer from README.md to CONTRIBUTING.md where users can find this table. Let me know what you think about these changes.

Reply:
I noticed there is also a similar table in `MAINTAINERS.md` but I suppose they are not exactly the same. That looks fine to me.